apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -1035,28 +1057,20 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
     };
 
     return (
-        application && !isInboundProtocolsRequestLoading && inboundProtocolList != undefined
-        && (tabPaneExtensions || !applicationConfig.editApplication.extendTabs
-            || application?.templateId === ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC
-            || application?.templateId === ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
-            || application?.templateId === ApplicationManagementConstants.CUSTOM_APPLICATION_SAML 
-            || application?.templateId === ApplicationManagementConstants.MOBILE)
-            ? (
-                <>
-                    <ResourceTab
-                        activeIndex= { activeTabIndex }
-                        data-testid= { `${testId}-resource-tabs` }
-                        defaultActiveIndex={ defaultActiveIndex }
-                        onTabChange={ handleTabChange }
-                        panes= { resolveTabPanes() }
-                        onInitialize={ ({ panesLength }) => {
-                            setTotalTabs(panesLength);
-                        } }
-                    />
-                    { showClientSecretHashDisclaimerModal && renderClientSecretHashDisclaimerModal() }
-                </>
-            )
-            : <ContentLoader />
+        <>
```

Comment:
Redundant fragment.
```suggestion
```

Reply:
Getting an error as `{ showClientSecretHashDisclaimerModal && renderClientSecretHashDisclaimerModal() }` is there in line 1072 after the `<ResourceTab>`.